### PR TITLE
fix: updated logic which was using deprecated lookup function

### PR DESCRIPTION
**Pull request by akocbek**

### CODE REVIEW

File: patterns/mixed/variables.tf [hunk 1]
```diff
@@ -190,6 +190,7 @@ variable "entitlement" {
   default     = null
 }
 
+# tflint-ignore: terraform_unused_declarations
```
Comment (ocofaigh) [inline patterns/mixed/variables.tf:193]: why are we ignoring this? If it is not used, it should not be here, but why is it not used?

File: patterns/roks/module/variables.tf [hunk 2]
```diff
@@ -136,6 +136,7 @@ variable "workers_per_zone" {
   default     = 1
 }
 
+# tflint-ignore: terraform_unused_declarations
```
Comment (ocofaigh) [inline patterns/roks/module/variables.tf:139]: why are we ignoring this? If it is not used, it should not be here, but why is it not used?

File: variables.tf [hunk 3]
```diff
@@ -682,6 +682,7 @@ variable "cos" {
 # Service Instance Variables
 ##############################################################################
 
+# tflint-ignore: terraform_unused_declarations
```
Comment (ocofaigh) [inline variables.tf:685]: why are we ignoring this? If it is not used, it should not be here, but why is it not used?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@akocbek we should not be ignoring any `terraform_unused_declarations` - we need to find out why they are unused and either use them, or remove them.

File: patterns/mixed/variables.tf (hunk 1 above)
Comment (akocbek) [reply patterns/mixed/variables.tf]: it is not used inside patterns, instead the value is take from override our default is set. Will remove it

File: variables.tf (hunk 3 above)
Comment (akocbek) [reply variables.tf:685]: it is not used inside patterns, instead the value is take from override our default is set. Will remove it

File: patterns/mixed/variables.tf (hunk 1 above)
Comment (akocbek) [reply patterns/mixed/variables.tf]: fixed.

File: patterns/roks/module/variables.tf (hunk 2 above)
Comment (akocbek) [reply patterns/roks/module/variables.tf]: fixed

**Review by Aashiq-J — APPROVED** [review]
Looks good to me

**Comment by ocofaigh** [comment]
The `TestRunVPCPatternSchematics` test failed with:
```
2023/12/12 10:29:31 Terraform apply | Error: [ERROR] Error in getting bucket info rule: Get "https://config.cloud-object-storage.cloud.ibm.com/v1/b/ocp-sc-cmjyi2-atracker-bucket-e5pwm8i8": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```
Retrying..

**Pull request merged**